src/index.js

```diff
@@ -83,8 +96,9 @@ closePopUp.addEventListener('click', () => {
   commentTable.innerHTML = '';
 });
 // CALL FUNCTIONS
+renderLikes().then((response) => getPokemonInfo(response));
 
-getPokemonInfo();
+// getPokemonInfo();
```

Comment:
- [ ] We can get rid of this commented line as it is already implemented inside the anonymous function